### PR TITLE
[DDW-502]  Enable EKG server for non-mainnet networks

### PR DESCRIPTION
**Pull request by deepfire**
This enables `--ekg-server localhost:808{1,2,3}` for `testnet`/`staging`/`demo` clusters.

The EKG dash looks like: 

screen shot 2018-11-23 at 13 28 24

...and is accessible from the Network status screen:

screen shot 2018-11-23 at 13 43 58

---

### Basics

- [ ] PR is updated to the most recent version of target branch (and there are no conflicts)
- [ ] PR has good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance tests are passing (`yarn run test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn run dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn run package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn run flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn run lint`)
- [ ] Text changes are proofread and approved (Jane Wild)
- [ ] There are no missing translations (running `yarn run manage:translations` produces no changes)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn run storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly documented and commented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-rendering
- [ ] Any code that only works in Electron is neatly separated from components

### Testing
- [ ] New feature / change is covered by acceptance tests
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature / change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review:
- [ ] Merge PR
- [ ] Delete source branch
- [ ] Move ticket to `done` on the Youtrack board

### CODE REVIEW

**Comment by nikolaglumac** [comment]
@deepfire do we need this for 0.12.0 release? Or is it just for debugging sync-stalling issues?

**Comment by deepfire** [comment]
@nikolaglumac, I think both points are true.

1. Yes this is intended for sync stalling issues.
2. This is still a useful feature to have in general, to diagnose `cardano-node` resource usage.
3. This will not affect the `mainnet` wallet at all, since the server won't be started.

**Comment by nikolaglumac** [comment]
Thanks @deepfire - please keep in mind that we need to be careful so that there are no collisions between Staging and Testnet Daedalus instances while running in parallel 🙏

**Comment by deepfire** [comment]
Yay, there we go: https://imgur.com/xvW9SKU

**Comment by deepfire** [comment]
https://github.com/input-output-hk/daedalus/pull/1188/files#diff-0aa3d3ba86cf1233a4a682208fb1c38cR7  is where I got stuck -- can't define a route..

File: installers/dhall/launcher.dhall [hunk 1]
```diff
@@ -26,5 +26,5 @@
     , "--wallet-address",      "127.0.0.1:0"
     -- XXX: this is a workaround for Linux
     , "--update-with-package"
-    ]
+    ] # cluster.extraNodeArgs
```
Comment (cleverca22) [inline installers/dhall/launcher.dhall]: is # a comment in dhall or the operator to concat 2 list?

**Pull request merged**